src/types/rook.io/v1alpha1/volume.ts

```diff
@@ -0,0 +1,11 @@
+export interface LocalObjectReference {
+    name: string;
+}
+
+export interface FlexVolumeSource {
```

Comment:
The `FlexVolumeSource` belongs to the `core/v1` API group ([ref](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/#flexvolumesource-v1-core)).